docs/es-overview.asciidoc

```diff
@@ -2,169 +2,41 @@
-protects your hosts <<malware-prevention, against malware>> and ships these data sets:
-***  *Windows*: Process, network, file, DNS, registry, DLL and driver loads,
-malware security detections, API
-*** *Linux/macOS*: Process, network, file
-** {integrations-docs}[{integrations}]: Integrations are a streamlined way to send your data to the {stack}. Integrations are available for popular services and platforms, like Nginx, AWS, and MongoDB, as well as many generic input types like log files.
-** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
-are lightweight data shippers. Beat modules provide a way of collecting and
-parsing specific data sets from common sources, such as cloud and OS events,
-logs, and metrics. Common security-related modules are listed
-<<enable-beat-modules, here>>.
-* The {security-app} in {kib} is used to manage the *Detection engine*,
-*Cases*, and *Timeline*, as well as administer hosts running {elastic-defend}:
-** Detection engine: Automatically searches for suspicious host and network
-activity via the following:
-*** <<detection-engine-overview, Detection rules>>: Periodically search the data
-({es} indices) sent from your hosts for suspicious events. When a suspicious
-event is discovered, an alert is generated. External systems, such as
-Slack and email, can be used to send notifications when alerts are generated.
-You can create your own rules and make use of our <<prebuilt-rules, prebuilt ones>>.
-*** <<detections-ui-exceptions, Exceptions>>: Reduce noise and the number of
-false positives. Exceptions are associated with rules and prevent alerts when
-an exception's conditions are met. *Value lists* contain source event
-values that can be used as part of an exception's conditions. When
-{elastic-defend} is installed on your hosts, you can add malware exceptions
-directly to the endpoint from the Security app.
-*** <<included-jobs, {ml-cap} jobs>>: Automatic anomaly detection of host and network events. Anomaly scores are provided per host and can be used with detection rules.
-** <<timelines-ui, Timeline>>: Workspace for investigating alerts and events.
-Timelines use queries and filters to drill down into events related to
-a specific incident. Timeline templates are attached to rules and use predefined
-queries when alerts are investigated. Timelines can be saved and shared with
-others, as well as attached to Cases.
-** <<cases-overview, Cases>>: An internal system for opening, tracking, and sharing
-security issues directly in the Security app. Cases can be integrated with
-external ticketing systems.
-** <<admin-page-ov, Administration>>: View and manage hosts running {elastic-defend}.
+[[siem-integration]]
+=== Learn more
 
-<<ingest-data>> and <<install-endpoint>> describe how to ship security-related
-data to {es}.
+* <<getting-started, Get started>>: System requirements, workspaces, configuration, and data ingestion.
```

Comment:
```suggestion
* <<getting-started, Get started>>: Learn about system requirements, workspaces, configuration, and data ingestion.
```
Minor suggestion for consistency, since all other bullets in this section start with a verb.